curriculum/challenges/english/25-front-end-development/workshop-storytelling-app/671fa769540d152e53327f4b.md

```diff
@@ -0,0 +1,147 @@
+---
+id: 671fa769540d152e53327f4b
+title: Step 7
+challengeType: 0
+dashedName: step-7
+---
+
+# --description--
+
+Select the `#scary-btn`, `#funny-btn`, and `#adventure-btn` buttons and store them in variables called `scaryStoryBtn`, `funnyStoryBtn`, and `adventureStoryBtn`.
+
+# --hints--
+
+Your `scaryStoryBtn` variable should have the value of `document.getElementById("scary-btn")`.
+
+```js
+assert.deepEqual(scaryStoryBtn, document.getElementById("scary-btn"));
+```
+
+Your `funnyStoryBtn` should have the value of `document.getElementById("funny-btn")`.
+
+```js
+assert.deepEqual(funnyStoryBtn, document.getElementById("funny-btn"));
+```
+
+Your `adventureStoryBtn` should have the value of `document.getElementById("adventure-btn")`.
+
+```js
+assert.deepEqual(adventureStoryBtn, document.getElementById("adventure-btn"));
+```
```

Comment:
The directions don't mention that you have to use `getElementById`. but `querySelector` is also a valid answer.
The hints need to be updated to account for that.

Also, the tests should be more granular. 
We should have tests to check that each of the variables exists, then check that the correct value has been assigned to it.

Reply:
Done!